rsp/00x_bz_rest_api.md

```diff
@@ -0,0 +1,159 @@
+# RSP - 1 — *bz* REST API
+
```

Comment:
I think you should mention that everything should be prefixed by something like `/api/v1/bz`. If you agree to this convention (obviously). Note that @Vaelden added this to the top of his RSP for the lycan API.